eks-worker-al2.json

```diff
@@ -8,11 +8,13 @@
     "arch": null,
     "associate_public_ip_address": "",
     "aws_access_key_id": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_cli_version": "2.4.10",
```

Comment:
Why does this need to be set? Are you just forcibly be upgrading it?

Reply:
This is set to match the current v2 CLI used. I don't see any reason to necessarily keep it at v2.4.10. But was just following the prior art. 

We could make it auto-update with no parameterization, although it may be useful to pin it depending on the CLI version release timings across partitions.

Reply:
good with me, removed version parameter so we always pull the latest.